### ISSUE TITLE
[Usage]: How to run vllm with regression task, just like classify task

### ISSUE THREAD

**Issue by zhanpengjie** [vllm-project/vllm#12379]
### Your current environment

I have trained qwen2.5 with regression task,  below is inference code : 

from transformers import AutoTokenizer, AutoModelForSequenceClassification
import torch

tokenizer = AutoTokenizer.from_pretrained("qwen3_score_0123", trust_remote_code=True)

model = AutoModelForSequenceClassification.from_pretrained(
    "qwen3_score_0123",
    num_labels=1,
    problem_type="regression",
    trust_remote_code=True
).to(torch.bfloat16)

device = torch.device("cuda:2" if torch.cuda.is_available() else "cpu")
model.to(device)
model.eval()

text = """
hello
"""

inputs = tokenizer(
    text,
    truncation=True,
    padding='max_length',
    max_length=512,
    return_tensors="pt"
)

inputs = {key: value.to(device) for key, value in inputs.items()}

with torch.no_grad():
    outputs = model(**inputs)

predicted_score = outputs.logits.item() 

### How would you like to use vllm

I want to inference the qwen llm regression model with vllm, need help ,thanks

### Before submitting a new issue...

- [x] Make sure you already searched for relevant issues, and asked the chatbot living at the bottom right corner of the [documentation page](https://docs.vllm.ai/en/latest/), which can answer lots of frequently asked questions.

**Comment by DarkLight1337** [comment]
You can set a custom pooler as explained here: https://docs.vllm.ai/en/latest/models/pooling_models.html
